torch/profiler/_utils.py

```diff
@@ -96,7 +96,7 @@ def __init__(self, prof: profile):
         self.profile = prof
         self.metrics: Dict[EventKey, EventMetrics] = {}
         self.compute_self_time()
-        self.event_keys = sorted((e for e in self.metrics.keys()),
+        self.event_keys = sorted((e for e in self.metrics),
```

Comment:
```suggestion
        self.event_keys = sorted(self.metrics,
```

Reply:
Done

Reply:
I just opened an issue to add a new flake8 comprehension rule to detect these. :)

Reply:
There probably is one already IIRC. I am not sure if it is enabled in our config

Reply:
@justinchuby Nah, I enabled almost every flake8-comprehension rule in our config, even the ones I added recently. :)